rfq/order.go

```diff
@@ -80,13 +80,15 @@ type Policy interface {
 // AssetSalePolicy is a struct that holds the terms which determine whether an
 // asset sale channel HTLC is accepted or rejected.
 type AssetSalePolicy struct {
-	// ID is the unique identifier of the RFQ session that the policy is
-	// associated with.
-	ID rfqmsg.ID
+	// AcceptedQuoteId is the unique identifier of the RFQ session quote
+	// accept message that the policy is associated with.
+	AcceptedQuoteId rfqmsg.ID
 
-	// MaxAssetAmount is the maximum amount of the asset that is being
-	// requested.
-	MaxAssetAmount uint64
+	// MaxOutboundAssetAmount represents the maximum asset amount permitted
+	// by policy for outbound transactions. It sets an upper limit on the
+	// amount of assets this node is willing to divest within the remit of
+	// the policy.
+	MaxOutboundAssetAmount uint64
```

Comment:
IMO this name is a bit confusing. Since in the RFQ doc and on the wire we actually talk about the `max_in_asset`... 

In my mind it would be more consistent if this was called `MaxInAsset` here, the asset buy policy as well as in the `BuyRequest` and `SellRequest` messages.

OR do the switch at the transition from wire to request struct, but then have the units correct on the request structs (one being `uint64` and the other being `lnwire.MilliSatoshi`).
And the policies should 100% match the request structs (both in name and unit type), otherwise it'll be even more confusing.

Reply:
`order.go` is completely from the point of view of an edge node. That's one reason why we're using "policy" structs here and copying over fields rather than just carrying along an entire accepted quote.

I think things would be more confusing if we write everything from a wallet node's PoV including the edge node only routines.